x11-packages/wlroots/0006-fix-vulkan-renderer-build.patch

```diff
@@ -0,0 +1,207 @@
+diff --git a/render/vulkan/meson.build b/render/vulkan/meson.build
+index 0c45a713..f7db70ac 100644
+--- a/render/vulkan/meson.build
++++ b/render/vulkan/meson.build
+@@ -24,18 +24,6 @@ if not cc.check_header('vulkan/vulkan.h', dependencies: dep_vulkan)
+ 	endif
+ endif
+ 
+-glslang = find_program('glslangValidator', native: true, required: false)
+-if not glslang.found()
+-	if 'vulkan' in renderers
+-		error('\n'.join(msg).format('glslang'))
+-	else
+-		subdir_done()
+-	endif
+-endif
+-
+-glslang_version_info = run_command(glslang, '--version', check: true).stdout()
+-glslang_version = glslang_version_info.split('\n')[0].split(':')[-1]
+-
+ wlr_files += files(
+ 	'renderer.c',
+ 	'texture.c',
+@@ -46,5 +34,3 @@ wlr_files += files(
+ 
+ wlr_deps += dep_vulkan
+ features += { 'vulkan-renderer': true }
+-
+-subdir('shaders')
+diff --git a/render/vulkan/shaders/common.vert.h b/render/vulkan/shaders/common.vert.h
+new file mode 100644
+index 00000000..8cdadf7c
+--- /dev/null
++++ b/render/vulkan/shaders/common.vert.h
+@@ -0,0 +1,57 @@
++	// 1114.1.0
++	 #pragma once
++const uint32_t common_vert_data[] = {
```

Comment:
You could add `glslang-tools` in `./scripts/setup-ubuntu.sh`, and then I suppose this could be generated automatically by the building system.